go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -428,6 +437,24 @@ func (c *addressManager) sync() {
 	}
 }
 
+// getNonOVNEgressIPs returns the set of egress IPs that are assigned to non-OVN managed interfaces. The
+// addresses are used to support Egress IP multi NIC feature. The addresses must not be included in address manager
+// because the addresses are only to support Egress IP multi NIC feature and must not be exposed via host-cidrs annot.
+func (c *addressManager) getNonOVNEgressIPs() (sets.Set[string], error) {
```

Comment:
getSecondaryHostInterfaceEgressIPs

its a mouthful, but at least it is more descriptive of what these things *are* rather then what they *are not*

Reply:
Ya, I rather that approach too. I just could not come up with something to describe it that also excluded secondary networks created by OVN and the possibility of folks thinking an EIP could be assigned there too. Lets chat on slack.